api/src/main/java/org/jboss/forge/roaster/model/Type.java

```diff
@@ -34,6 +38,8 @@
 
    boolean isQualified();
 
+   boolean isNameQualified();
```

Comment:
It looks like the isQualified implementation should be changed instead of introducing this new method